.gitignore

```diff
@@ -0,0 +1,4 @@
+**/.idea
+**/__pycache__
+
```

Comment:
Please add database files (sqlite3) to gitignore as well.

Generally, it'd be better to use the ready-to-use templates of gitignore files (examples could be found herehttps://github.com/github/gitignore)

Reply:
fixed